trail/upload/process_uploads/processors/lbt_processor.py

```diff
@@ -0,0 +1,102 @@
+"""
+Class for processing FITS files processed by Large Binocular Telescope.
+
+These pipelines will bundle entire focal planes into a single file, which can
+be successfully processed by the MultiExtensionFits class, but for which we can
+create better visualisations.
+"""
+
+
+from PIL import Image
+import numpy as np
+
+from .multi_extension_fits import MultiExtensionFits
+from upload.models import Thumbnails
+
+
+__all__ = ["LbtFits", ]
+
+
+class LbtFits(MultiExtensionFits):
+
+    name = "LbtFits"
+    priority = 2
+
+    def __init__(self, uploadInfo, uploadedFile):
+        super().__init__(uploadInfo, uploadedFile)
+        # Override the default processed exts to filter only science images
+        # from all image-like exts, ignoring focus and guider chips.
+        self.exts = self._getScienceImages(self.exts)
+
+    @classmethod
+    def _getScienceImages(cls, hdulist):
+        exts = []
+        for hdu in hdulist:
+            exttype = hdu.header.get("XTENSION", False)
+            if exttype.strip() == 'IMAGE':
+                exts.append(hdu)
+
+        return exts
+
+    @classmethod
+    def canProcess(cls, uploadedFile, returnHdulist=False):
+        canProcess, hdulist = super().canProcess(uploadedFile, returnHdulist=True)
+        origin = hdulist[0].header.get("ORIGIN", False)
+        if origin and "LBT Observatory" == origin:
+            return True
+        return False
+
+    def _createFocalPlaneImage(self, focalPlane):
+        for ext in self.exts:
+            # no matter how painful this is, if we don't, normalize will mutate
+            # in science data in place....
+            image = ext.data.copy()
+            image = self.normalizeImage(image)
+
+            # TODO: test here if the step-vise resizing is faster...
+            image = image.resize((focalPlane.scaledY, focalPlane.scaledX),
+                                 Image.ANTIALIAS)
+
+            focalPlane.add_image(image, ext.header["DETPOS"])
+
+        return focalPlane
+
```

Comment:
This is the DECam processor method we copied here for reference. It isn't used anywhere in the code so remove it.